docs/api_auth.md

```diff
@@ -0,0 +1,35 @@
+# Relay API Authentication
+The Relay API is built on [Django REST Framework][drf] and authenticates
+requests with any of 3 methods:
+* [FXA OAuth Token Authentication](fxa-oauth-token-authentication)
+* [`SessionAuthentication`][SessionAuthentication]
+* [`TokenAuthentication`][TokenAuthentication]
+
+## FXA OAuth Token Authentication
+The Relay add-on uses the [`identity.launchWebAuthFlow` API][mdn-webauthflow]
+to perform an OAuth2 flow with [the FXA OAuth service][fxa-oauth], including
+[PKCE][fxa-pkce].
+
+After the OAuth flow is complete, the add-on has an FXA access token and a
+long-living FXA refresh token, and authenticates all requests to the Relay
+server by including an `Authorization: Bearer {fxa-access-token}` header in all
+API requests. The Relay server checks the token against [the FXA OAuth `/verify`
+endpoint][fxa-oauth-token-verify].
+
+This auth scheme can be used by other clients too. E.g., Firefox browser has a
+[`getOAuthToken`][searchfox-getoauthtoken] function which can be used to
+perform the same API authentication as the add-on.
+
+![FXA OAuth Token Authentication Sequence Diagram](api_auth.png)
+
+### TODOs
+* Cache the token verification check so we don't hit FXA on every client request.
```

Comment:
Did we not already do this [here](https://github.com/mozilla/fx-private-relay/pull/1514/files#diff-26e3f029d42ad868ed2e12764aa9e1263473a75b7d2125d3930de69e18665febR58)?